root/ directory is needed by profile.mk, even if there is no need for it

If you compile a profile without the `root/` folder, like the [libremesh/suggested-packages](https://github.com/libremesh/network-profiles/tree/master/libremesh/suggested-packages) one, you get an error like:

```
cp -fpR -r ./root/* /home/ilario/software/openwrt19/build_dir/target-mipsel_24kc_musl/profile-libremesh-suggested-packages/.pkgdir/profile-libremesh-suggested-packages/
cp: cannot stat './root/*': No such file or directory
make[3]: *** [../../profile.mk:64: /home/ilario/software/openwrt19/build_dir/target-mipsel_24kc_musl/profile-libremesh-suggested-packages/.pkgdir/profile-libremesh-suggested-packages.installed] Error 1
```

adding an empty `root/` directory is an annoying thing to do in git, so it would be better if profile.mk could check if the root directory exists here:

https://github.com/libremesh/network-profiles/blob/78d4c083245fa9720662bd4aa6e2ce1f5c62d4c7/profile.mk#L49-L52

--- a/profile.mk
+++ b/profile.mk
@@ -48,7 +48,8 @@ endif
 
 define Package/$(PKG_NAME)/install
 	$(INSTALL_DIR) $(1)/
-	$(CP) -r ./root/* $(1)/
+	# check if root/ directory exists before copying
+	if [ -e ./root/* ]; then $(CP) -r ./root/* $(1)/; fi
 endef
 
 define Build/Compile

